dart/dynamics/BodyNode.cpp

```diff
@@ -1339,6 +1339,14 @@ void BodyNode::notifyTransformUpdate()
 {
   notifyVelocityUpdate(); // Global Velocity depends on the Global Transform
 
+  // Jacobian calculations are dependent on the parent's world transform, but
+  // not on the world transform of their own BodyNode, so they must be dirtied
+  // regardless of whether the world transform of this BodyNode is already dirty
```

Comment:
This sounds good to me! These updates might be able to be applied to `osgDart` branch or master once the branch is merged since `JacobianNode` is introduced by `osgDart` branch.

In addition, we might need `notifyJacobianDerivUpdate()`, and it should be called by `BodyNode::notifyVelocityUpdate()`